src/pages/TermsOfService.test.ts

```diff
@@ -61,7 +92,8 @@ describe('TermsOfService', () => {
     // Assert
     expect(getTosFn).toHaveBeenCalled();
 
-    screen.getByText('some text');
+    const tosText = screen.getByText('some text');
+    expect(tosText).toBeInTheDocument();
```

Comment:
FYI, `getBy...` queries throw an error if there's no matching element found, so just the query is sufficient to test.

```suggestion
    screen.getByText('some text');
```